ws/ws.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/codequest-eu/octopussy"
+)
+
+func startServer(amqpUrl string) *octopussy.Server {
+	server := &octopussy.Server{
+		// TODO(janek): move to env variable?
+		Exchange: "ws_notification",
+		OnError: func(e error) {
+			log.Fatal(e.Error())
+		},
+		OnConn: func(r *http.Request) {
+			log.Println("Connecting " + r.Host)
+		},
+	}
+	if err := server.Dial(amqpUrl); err != nil {
+		return nil
+	}
+	return server
+}
+
+func main() {
+	flag.Parse()
+	addr := flag.String("addr", ":9090", "service address")
```

Comment:
Consider using [this library](https://github.com/codegangsta/cli) for configuration.